api/create_tables.sql

```diff
@@ -0,0 +1,56 @@
+CREATE DATABASE IF NOT EXISTS cs348db;
+USE cs348db;
+
+CREATE TABLE IF NOT EXISTS UnitListerAccount(
+    pm_id int NOT NULL AUTO_INCREMENT,  
+    name VARCHAR(255) NOT NULL,
+    phone_num VARCHAR(255) NOT NULL,
```

Comment:
We might want to restrict the number of characters on this, since phone numbers in Canada are generally 10 characters.
```suggestion
    phone_num CHAR(10) NOT NULL,
```

Reply:
👍